docs/en/integrations/data-visualization/explo-and-clickhouse.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_label: Explo
+sidebar_position: 131
+slug: /en/integrations/explo
+keywords: [clickhouse, Explo, connect, integrate, ui]
+description: Explo is an easy-to-use, open source UI tool for asking questions about your data.
+---
+import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
+
+# Connecting Explo to ClickHouse
+
+Customer-facing analytics for any platform. Designed for beautiful visualization. Engineered for simplicity.
+
+## Goal
+
+In this guide you will connect your data from ClickHouse to Explo and visualize the results.  The chart will look like this:
+<img src={require('./images/explo_15.png').default} class="image" alt="Explo Dashboard" />
+
+<p/>
+
+:::tip Add some data
+If you do not have a dataset to work with you can add one of the examples.  This guide uses the [UK Price Paid](/docs/en/getting-started/example-datasets/uk-price-paid.md) dataset, so you might choose that one.  There are several others to look at in the same documentation category.
+:::
+
+## 1. Gather your connection details
+<ConnectionDetails />
+
+
+## 2.  Connect Explo to ClickHouse
+
+1. Sign up for an Explo account.
+
+2. Click on the Explo **data** tab on the left hand sidebar.
+
+
+<img src={require('./images/explo_01.png').default} class="image" alt="Data Tab" />
+
+3. Click **Connect Data Source** in the upper right hand side.
+
+
+<img src={require('./images/explo_02.png').default} class="image" alt="Connect Data Source" />
+
+4. Fill out the information on the **Getting Started** page
+
+
+<img src={require('./images/explo_03.png').default} class="image" alt="Getting Started" />
+
+5. Select **Clickhouse**
+
+
+<img src={require('./images/explo_04.png').default} class="image" alt="Clickhouse" />
+
+
+6. Enter your **Clickhouse Credentials**. 
+
+
+<img src={require('./images/explo_05.png').default} class="image" alt="Credentials" />
+
+
+7. Configure **Security**, and you are done!
```

Comment:
Could you provide a link to a page with all the Explo egress IPs? These IP must be whitelisted in ClickHouse Cloud

Reply:
Done!